src/main/java/org/opensearch/knn/plugin/stats/StatNames.java

```diff
@@ -44,6 +44,7 @@
     INDICES_IN_CACHE("indices_in_cache"),
     CIRCUIT_BREAKER_TRIGGERED("circuit_breaker_triggered"),
     MODEL_INDEX_STATUS("model_index_status"),
+    MODEL_INDEXING_DEGRADED("model_indexing_degraded"),
```

Comment:
I think I rather call this index_from_model_degraded. model_indexing_degraded may be confused with indexing a model is degraded.

Reply:
Ack